packages/firestore/src/model/target_index_matcher.ts

```diff
@@ -110,6 +114,11 @@ export class TargetIndexMatcher {
       'Collection IDs do not match'
     );
 
+    if (this.hasMultipleInequality()) {
+      // Only single inequality is supported for now.
```

Comment:
Change this comment to something like "TODO(b/NNNNNN) Add support for more than one inequality"